html/semantics/text-level-semantics/the-a-element/a-download-click-404.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Clicking on an &lt;a> element with a download attribute and href that leads to 404 should not navigate</title>
+<link rel="author" title="phistuck" href="mailto:user@domain.com">
```

Comment:
Is this your actual email? (An author `link` is not required; that info is in the git log anyway.)